evaluation.md

```diff
@@ -1,19 +1,68 @@
 # Evaluation: running commentary
 
-## Internal DSL
+Please see [reflection.md](/reflection.md) for additional comments on our work.
 
-_Describe each change from your ideal syntax to the syntax you implemented, and
-describe_ why _you made the change._
+## Internal DSL
 
 **On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
 
+3. Our ideal syntax isn't too different from the internal DSL we wrote. The main 
+features such as going to a certain surrounding, jumping from a label to the other, 
+and facing a specific direction remain the same. There are only two main differences.
+Firstly, we were not able to write an `until` method so that instead of saying
+`Go(<surrounding>)` we could say `Go until <surrounding>`. Secondly, to translate the internal
+AST into the provided API we realized we needed to have our `Label` take in a list of rules
+and that we needed a wrapper case class that took in a list of `Label`s. This meant we
+had to have a program structure like:
+```
+Program (
+  List(
+    Label("one"){
+      List(
+        Rule1,
+        Rule2,
+        Rule3
+      )
+    },
+    Label("two"){}
+  )
+)
+```
+instead of the intended internal structure of:
+```
+Label("one"){
+    Rule1
+    Rule2
+}
+Label("two"){
+    Rule3
+}
+
+
 **On a scale of 1–10 (where 10 is "very difficult"), how difficult was it to map your syntax to the provided API?**
+10. We never actually succeeded so it was indeed very difficult to us (as better explained in the `reflection.md`
+file). We felt that the reason it was very difficult was because to have a `Jump` feature, we had to know the
+start states for every `Label`. This not only required parsing through the input twice to map our syntax to
+the provided API but also required a scala `Map[String,Int]` and some other class variables, which eventually
+caused the errors that halted our progress. We think we eventually figured out how to map each part of our syntax
+to the provided API though!
+
```

Comment:
I agree this is very difficult. It would have been better to complete something that works and go from there.